src/chrome/content/rules/GameSpot.xml

```diff
@@ -1,86 +1,12 @@
 <!--
-
-	For rules causing false/broken MCB, see GameSpot.com-falsemixed.xml.
-
 	For other CBS coverage, see CBS.xml.
-
-
-	CDN buckets:
-
-		- static.gamespot.com.edgesuite.net
-
-			- a50.g.akamai.net
-
-
-	Nonfunctional domains:
-
-		- userimage.gamespot.com	(↓ ditto)
-		- im(age|g[12].gamespotcdn.net	(Akamai; "An error occurred"; all equivalent)
-
-
-	Problematic subdomains:
-
-		- static *
-		- static[1-4] *
-		- www ²
-
-	* Works, akamai
-	² Mixed css from static1
-
-
-	Fully covered subdomains:
-
-		- ^
-		- auth
-
-
-	Observed cookie domains:
-
-		- ^ ¹
-		- . ²
-		- auth ³
-		- www ¹
-
```

Comment:
The comments  above are probably still useful to keep around fro future updates. Unless there was some reason that they'd only add to further confusion.

Reply:
@zoracon Done.